test/integration/self-hosted-notary/notary-service-container/server/config.json

```diff
@@ -1,17 +1,17 @@
 {
     "server": {
         "http_addr": ":4443",
-        "tls_key_file": "./notary-server.key",
-        "tls_cert_file": "./notary-server.crt"
+        "tls_key_file": "./notary.key",
+        "tls_cert_file": "./notary.crt"
     },
     "trust_service": {
         "type": "remote",
         "hostname": "notary.signer",
         "port": "7899",
-        "tls_ca_file": "./ca.crt",
+        "tls_ca_file": "./notary.crt",
         "key_algorithm": "ecdsa",
-        "tls_client_cert": "./notary-server.crt",
```

Comment:
ca and cert are now the same aye?

Reply:
Yes. Thought it's easier to not have a dedicated root, but rather have self-signed cert